daemon/cmd/daemon_test.go

```diff
@@ -312,6 +320,18 @@ func (ds *DaemonSuite) RemoveIdentity(id *identity.Identity) {}
 
 func (ds *DaemonSuite) RemoveOldAddNewIdentity(old, new *identity.Identity) {}
 
+func (ds *DaemonSuite) PolicyAdd(rules policyAPI.Rules) {
```

Comment:
Since `PolicyAdd()` prototype alraedy changed, maybe rename this as `PolicyImport()`?

Reply:
sure, and unexported, since these are just convenience wrappers.